Makefile

```diff
@@ -141,6 +153,52 @@ liccheck:
 copyright-check:
 	python scripts/check_copyright.py
 
+####################
+### Clean and init commands
+####################
+
+.PHONY: rmcache
+rmcache:
+	find . | grep -E "\(__pycache__|\.pyc|\.pyo$)" | xargs rm -rf
+
+# Run outside Pipenv
+# To exit Pipenv run "deactivate" or Control+D
+.PHONY: clean
+clean:
+	rm -rf .tox
+	rm -rf arcturus.egg-info
+	rm -rf cosmpy.egg-info
+	rm -rf src/cosm.egg-info
+	rm -rf reqlib-metadata
+	rm -rf coverage_report
+	rm -rf .mypy_cache
+	rm -rf .pytest_cache
+	rm .coverage
+	make rmcache
+	pipenv --rm
+
+.PHONY: create
+create:
+	pipenv install
+
+.PHONY: create-dev
+create-dev:
+	pipenv install --dev
```

Comment:
No need for these vanilla ones. Let's just have the options below with the clean.